### PR TITLE
Release prep 2.3.0

### PR DESCRIPTION
**Pull request by HelenCampbell**

### CODE REVIEW

**Comment by bmjen** [comment]
@HelenCampbell Ubuntu Xenial needs to be added to the metadata.json.

**Comment by bmjen** [comment]
@HelenCampbell PE block in metadata also needs to be removed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+## Supported Release 2.3.0
+### Summary
+A release containing many bugfixes with additional features, including a modulesync update.
+
+#### Features
+- Apt_updates facts now use /usr/bin/apt-get.
+- Addition of notify update to apt::source.
+- Update to newest modulesync_configs.
+- Installs software-properties-common for xenial.
+- Modulesync updates.
```
Comment (daenney) [inline CHANGELOG.md]: Do we need to mention module sync twice? Most people have no idea what it is / don't really care.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+- Add ability to specify a hash of apt::conf defines.
+
+#### Bugfixes
+- Update ke_compat_spec to run under STRICT_VARIABLES.
```
Comment (daenney) [inline CHANGELOG.md]: There's nothing named like this in the repo. What are we referring to?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+- A clean up of key_compat_specs.
```
Comment (daenney) [inline CHANGELOG.md]: There's nothing named like this in the repo. What are we referring to?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+- Apt::setting expects priority to be an integer, set defaults accordingly.
+- Fixed version check for Ubuntu on 16.04.
+- Now uses hkps.pool.sks-keyservers.net instead of pgp.mit.edu.
+- Updates and fixes to tests. General cleanup.
+- Fixed regexp for $ensure params.
+- Apt/params: Remove unused LSB facts.
+- Replaced `-s` with `-f` in ppa rspec tests.
```
Comment (daenney) [inline CHANGELOG.md]: Maybe we should explain what this does? Just a flag change is not very informative.

**Pull request merged**